onebusaway-android/src/main/java/org/onebusaway/android/report/ui/dialog/CustomerServiceDialog.java

```diff
@@ -123,11 +123,14 @@ protected void initView(View view, ObaAgencyWithCoverage coverage) {
                     public void onClick(View view) {
                         String locationString = getActivity().getIntent().
                                 getStringExtra(BaseReportActivity.LOCATION_STRING);
-                        String agencyName = "\nAgency: " + agency.getName();
-                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString + agencyName);
+                        UIUtils.sendEmail(getActivity(), agency.getEmail(), locationString);
+
+                        String gaEventLabel = locationString != null ?
```

Comment:
In other words, I'd prefer not to have to combine `analytics_label_customer_service_email` and `analytics_label_customer_service_email_without_location` to count how many customer service events there are.  I'd prefer to get this from `analytics_label_customer_service_email` directly, and then look at `analytics_label_customer_service_email_without_location` to know how many null location issues we've seen.  Might make sense to change the name of `analytics_label_customer_service_email_without_location` as well.